packages/holocron-dev-server/src/webpack/builds.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import webpack from 'webpack';
+
+import { createHolocronModuleWebpackConfig, createExternalsDllWebpackConfig } from './configs/bundles';
+import { modulesLibraryVarName } from './helpers';
+import { ufs } from '../utils';
+import {
+  error,
```

Comment:
Agree with the name, will update. We may not want to throw on every error since this is a tolerant development server, not every error is fatal to the process and some errors are recoverable.

Reply:
agreed, i've updated this